pkg/agent/baker_test.go

```diff
@@ -1350,15 +1350,20 @@ oom_score = 0
 				Expect(sysctlContent).To(ContainSubstring("net.ipv4.neigh.default.gc_thresh3=16384"))
 				Expect(sysctlContent).To(ContainSubstring("net.ipv4.ip_local_reserved_ports=65330"))
 			}),
-		Entry("AKSUbuntu2204 with SecurityProfile", "AKSUbuntu2204+SecurityProfile", "1.26.0",
+		Entry("AKSUbuntu2204 with bootstrap container registry enabled", "AKSUbuntu2204+BootstrapContainerRegistry", "1.30.0",
 			func(config *datamodel.NodeBootstrappingConfiguration) {
```

Comment:
no need to do dir renaming? can just use securityprofile. this helps make the real diff more clear.

we can do dir renaming in one dedicate PR later.

Reply:
kept securityProfile.